edward/util/random_variables.py

```diff
@@ -185,6 +185,10 @@ def copy(org_instance, dict_swap=None, scope="copied",
   else:  # tf.Operation
     op = org_instance
 
+    # Do not copy queue operations
```

Comment:
this makes sense. can you also update the docstring on line 26 to say that it also does not copy queue nodes? happy to merge it in afterwards.